HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,562 @@
+{
+    "SchemaDefinition": "RedfishInteroperabilityProfile.v1_6_0",
+    "ProfileName": "OCP Baseline Redfish Service",
+    "ProfileVersion": "0.6.0",
+    "Purpose": "Describes OCP requirements for any Redfish Service instance.  This profile covers only the purely software/firmware service portions of the Redfish schema.  Any device or product (software or firmware-based) implementing the Redfish Specification should meet these basic functionality requirements - regardless of the underlying hardware or system it is supporting.  This profile is intended to be included as a Required Profile by other OCP profiles to serve as a base for Redfish Service functionality.",
+    "OwningEntity": "OCP",
+    "ContactInfo": "jeff.autor@ocproject.com",
+    "Protocol": {
+        "MinVersion": "1.6",
+        "Discovery": "Recommended",
+        "HostInterface": "None",
+        "ExpandQuery": "Recommended",
+        "SelectQuery": "None",
+        "FilterQuery": "Recommended",
+        "OnlyQuery": "Mandatory",
+        "ExcerptQuery": "None",
+        "DeepPATCH": "None",
+        "DeepPOST": "None"
+    },
+    "Resources": {
+        "AccountService": {
+            "PropertyRequirements": {
+                "Accounts": {},
+                "ActiveDirecotory": {
+                    "ReadRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "ServiceEnabled": {}
+                    }
+                },
+                "Roles": {},
+                "LDAP": {
+                    "ReadRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "LDAPService": {
+                            "ReadRequirement": "IfImplemented"
+                        },
+                        "ServiceEnabled": {},
+                        "ServiceAddresses": {},
+                        "RemoteRoleMapping": {}
+                    }
+                },
+                "OAuth2": {
+                    "ReadRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "OAuth2Service": {},
+                        "ServiceEnabled": {}
+                    }
+                },
+                "OutboundConnections": {
+                    "ReadRequirement": "Recommended"
+                },
+                "SupportedAccountTypes": {}
+            }
+        },
+        "Certificate": {
+            "ReadRequirement": "Mandatory",
+            "URIs": "/redfish/v1/Managers/{ManagerId}/NetworkProtocol/HTTPS/Certificates/{CertificateId}",
+            "PropertyRequirements": {
+                "CertificateString": {}
+            }
+        },
+        "CertificateService": {
+            "ReadRequirement": "Mandatory",
+            "PropertyRequirements": {
+                "CertificateLocations": {}
+            },
+            "ActionRequirements": {
+                "GenerateCSR": {},
```

Comment:
I'd like "ActionInfo" as required here given the complexities of the action.

Reply:
I agree.